includes/Modules/Analytics_4/Settings.php

```diff
@@ -90,4 +92,27 @@ protected function get_sanitize_callback() {
 			return $option;
 		};
 	}
+
+	/**
+	 * Proxies setting values for select settings from classic Analytics.
+	 *
+	 * The values of these settings will be sourced from the original
+	 * Analytics module's settings, regardless of the value in the GA4 settings.
+	 *
+	 * @TODO Remove once Analytics and Analytics 4 modules are officially separated.
+	 *
+	 * @since n.e.x.t
+	 */
+	protected function proxy_ua_settings() {
+		$analytics_settings   = new Analytics_Settings( $this->options );
+		$proxy_values_from_ua = function ( $ga4_settings ) use ( $analytics_settings ) {
+			$ua_settings                     = $analytics_settings->get();
+			$ga4_settings['accountID']       = $ua_settings['accountID'];
+			$ga4_settings['adsConversionID'] = $ua_settings['adsConversionID'];
```

Comment:
@felixarntz As mentioned before, I think it would be preferable to encapsulate the common settings as much as possible here so we don't need to add so many special cases and todos elsewhere in the codebase.

Reply:
See above, let's not do this for now.